test/suites/vcf/resources/corner_cases.vcf

```diff
@@ -19,13 +19,13 @@
 ##FILTER=<ID=q40,Description="Quality below 40">
 ##FILTER=<ID=c5,Description="Coverage below 5">
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE0
-chr1	1701	.	T	C	93	PASS	DP=131	GT	0/1
-chr1	1821	.	G	T	93	PASS	DP=133	GT	0/1
-chr1	1977	.	A	G	93	PASS	DP=134	GT	0/1
-chr1	4013	.	T	C	93	PASS	DP=122	GT	0/1
+chr1	1701	.	T	<CNV:TR>,<CNV:TR>	93	PASS	DP=131	GT	0/1
```

Comment:
how about updating `corner_cases.vcf` with a `1/2` genotype for a SNV and a STR? this way we can check if the report works correctly.